src/app/models/question-dto.ts

```diff
@@ -1,40 +1,40 @@
 export class QuestionDto {
     public readonly title: string;
-    public readonly body: string;
+    public readonly questionBody: string;
     public readonly sampleAnswer: string;
     public readonly hints: string[];
     public readonly successRate: number;
     public readonly difficulty: string;
     public readonly parentTopicTitle: string;
     public readonly questionAnswerOptions: string[];
-    public solved: boolean;
+    public readonly answer: string;
 
     public constructor({
         title,
-        body,
+        questionBody,
         sampleAnswer,
         hints,
-        solved,
+        answer,
```

Comment:
I think we will still the need the solved attribute as it will allow us/the users the ability to filter based on whether a question hs been completed or not.